plugins/MumblePlugin.h

```diff
@@ -1518,6 +1519,35 @@ struct MUMBLE_API_STRUCT_NAME {
 
 	// -------- Request functions --------
 
+#	if SELECTED_API_VERSION >= MUMBLE_PLUGIN_VERSION_CHECK(1, 3, 0)
+
+	/**
+	 *
+	 * Requests Mumble to link the given channel to the given linkID.
+	 *
+	 * @param callerID The ID of the plugin calling this function
+	 * @param channelID The ID of the channel to link
+	 * @param linkID The ID of the base channel to link to
+	 * @returns The error code. If everything went well, STATUS_OK will be returned.
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *requestChannelLink)(mumble_plugin_id_t callerID,
+																		 mumble_channelid_t channelID,
+																		 mumble_channelid_t linkID);
+
+	/**
+	 *
+	 * Requests Mumble to unlink the given channel to the given unlinkID.
+	 *
+	 * @param callerID The ID of the plugin calling this function
+	 * @param channelID The ID of the channel to unlink
+	 * @param unlinkID The ID of the base channel to unlink to
+	 * @returns The error code. If everything went well, STATUS_OK will be returned.
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *requestChannelUnlink)(mumble_plugin_id_t callerID,
+																		   mumble_channelid_t channelID,
+																		   mumble_channelid_t unlinkID);
+#	endif
```

Comment:
I think these should be at the _end_ of the request functions block. This should make it easier for users to parse this header as most recent additions will be at the bottom of each group of functions.

Reply:
You mean that we can group functions depending on the API version. Am I right?

Reply:
Yes and no. The first grouping should be done by the kind of API functions (e.g. request functions) and then within a block of a given kind of functions, these functions should be ordered by the API version in which they have been introduced (where newest comes last)